.github/workflows/ci.yml

```diff
@@ -0,0 +1,52 @@
+name: CI
+on: [push, pull_request]
+jobs:
+
+  qmake:
+    name: "qmake on ${{ matrix.runner }}"
+    runs-on: "${{ matrix.runner }}"
+    strategy:
+      matrix:
+        runner:
+          - "ubuntu-16.04"
+          - "ubuntu-18.04"
+          - "ubuntu-20.04"
+    env:
+      DEBIAN_FRONTEND: noninteractive
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install qt and build tools
+        run: |
+          sudo apt-get update &&
+          sudo apt-get install -y --no-install-recommends \
+            pkg-config g++ make \
+            qt5-default qttools5-dev-tools
+      - name: Run qmake
+        run: mkdir build && cd build &&
+          qmake -r .. "QMAKE_CXXFLAGS=-std=c++11 -Werror"
+      - name: Build
+        run: cd build && make -j8
+
+  cmake:
+    name: "cmake on ${{ matrix.runner }}"
+    runs-on: "${{ matrix.runner }}"
+    strategy:
+      matrix:
+        runner:
+          - "ubuntu-16.04"
+          - "ubuntu-18.04"
+          - "ubuntu-20.04"
+    env:
+      DEBIAN_FRONTEND: noninteractive
+    steps:
+      - uses: actions/checkout@v1
+      - name: Install qt and build tools
+        run: |
+          sudo apt-get update &&
+          sudo apt-get install -y --no-install-recommends \
+            pkg-config g++ make cmake \
+            qt5-default qttools5-dev-tools
+      - name: Run cmake
```

Comment:
I'd add another step before this line:

    - name: Show cmake version
      run: cmake --version

Reply:
Done.